native/src/tablecloth.mli

```diff
@@ -728,34 +728,855 @@ module Char : sig
   val is_whitespace : char -> bool
 end
 
+module Float : sig
+  (** A module for working with {{: https://en.wikipedia.org/wiki/Floating-point_arithmetic } floating-point numbers}. Valid syntax for [float]s includes:
+    {[
+      0.
+      0.0
+      42.
+      42.0
+      3.14
+      0.1234
+      123_456.123_456
+      6.022e23   (* = (6.022 * 10^23) *)
+      6.022e+23  (* = (6.022 * 10^23) *)
+      1.602e−19  (* = (1.602 * 10^-19) *)
+      1e3        (* = (1 * 10 ** 3) = 1000 *)
+    ]}
+
+    Without opening this module you can use the [.] suffixed operators e.g
+    
+    {[ 1. +. 2. /. 0.25 *. 2. = 17. ]}
+
+    But by opening this module locally you can use the un-suffixed operators
+
+    {[Float.((10.0 - 1.5 / 0.5) ** 3.0) = 2401.0]}
```

Comment:
This is really cool!

Reply:
Unfortunately the example isn’t correct!